loops-and-arrays.js

```diff
@@ -27,6 +34,15 @@ const numbers = [2, 4, 5, 29, 38];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [ 6, 12, 15, 87, 114 ];
```

Comment:
Hierbij hetzelfde. Als ik het script wil runnen, krijg ik niet deze cijfers in mijn terminal.